libcodechecker/analyze/log_parser.py

```diff
@@ -26,14 +28,90 @@
     ['^-m(32|64)',
      '^-std=.*'])
 
+compiler_includes_dump_file = "compiler_includes.json"
+compiler_target_dump_file = "compiler_target.json"
 
-def get_compiler_includes(compiler, lang, compile_opts, extra_opts=None):
-    """
-    Returns a list of default includes of the given compiler.
-    """
+
+def get_compiler_err(cmd):
```

Comment:
Also, don't we have a method for arbitrary command invocation and output saving somewhere in `util`? Or am I mixing it up with the `test` library?

Reply:
there is `util.call_command` but that does not handle the stderr. Also this code is refactored from https://github.com/Ericsson/codechecker/pull/1050/files?diff=split#diff-3edc7a6cc15c42194786d15e8ab5852dL95